Classes/Command/FluidTemplatePrecompileCommand.php

```diff
@@ -0,0 +1,163 @@
+<?php
+namespace NamelessCoder\CmsFluidPrecompilerModule\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\Output;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use TYPO3Fluid\Fluid\Core\Cache\FluidCacheWarmupResult;
+
+/**
+ * A console command for precompiling Fluid templates
+ */
+class FluidTemplatePrecompileCommand extends Command
```

Comment:
Should be named `FluidPrecompileCommand` since it also warms up partials and layouts, see https://github.com/TYPO3/Fluid/blob/f583bad/src/Core/Cache/StandardCacheWarmer.php#L77-L79

Reply:
Agree!